asistente_ladm_col/gui/change_detection/dlg_change_detection_settings.py

```diff
@@ -55,10 +55,16 @@ def __init__(self, parent=None, qgis_utils=None, conn_manager=None):
         self._db_collected = self.conn_manager.get_db_connector_from_source()
         self._db_supplies = self.conn_manager.get_db_connector_from_source(SUPPLIES_DB_SOURCE)
 
-        # To postpone calling refresh gui until we close this dialog instead of settings
-        self._db_collected_was_changed = False
+        # There may be 1 case where we need to emit a db_connection_changed from the Export Data dialog:
+        #   1) Connection Settings was opened and the DB conn was changed.
+        self._db_collected_was_changed = False  # To postpone calling refresh gui until we close this dialog instead of settings
         self._db_supplies_was_changed = False
 
+        # Similarly, we could call a refresh on layers and relations cache in 1 case:
+        #   1) If the ED dialog was called for the COLLECTED source: opening Connection Settings and changing the DB
```

Comment:
"1) If the change detection settings dialog..."

Reply:
Solved in https://github.com/AgenciaImplementacion/Asistente-LADM_COL/pull/284/commits/d8cd32d73582bfbb5a2947422c7d845e02653db1